xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/Game.h

```diff
@@ -0,0 +1,2360 @@
+  /// Any attempts to connect a controller to a port on a disconnected multitap
+  /// will return false.
+  ///
+  virtual bool ConnectController(bool connect,
+                                 const std::string& port_address,
+                                 const std::string& controller_id)
+  {
+    return false;
+  }
+  //----------------------------------------------------------------------------
+
+  //============================================================================
+  ///
+  /// @brief Notify the add-on of an input event
+  ///
+  /// @param[in] event The input event
+  ///
+  /// @return true if the event was handled, false otherwise
+  ///
+  virtual bool InputEvent(const game_input_event& event)
+  {
+    return false;
+  }
+  //----------------------------------------------------------------------------
+
+  //============================================================================
+  ///
+  /// @brief **Callback to Kodi Function**<br>Notify the port of an input event
+  ///
+  /// @param[in] event The input event
+  /// @return true if the event was handled, false otherwise
+  ///
+  /// @note Input events can arrive for the following sources:
+  ///   - \ref GAME_INPUT_EVENT_MOTOR
+  ///
+  /// @remarks Only called from addon itself
+  ///
+  bool KodiInputEvent(const game_input_event& event)
+  {
+    return m_instanceData->toKodi.InputEvent(m_instanceData->toKodi.kodiInstance, &event);
+  }
+  //----------------------------------------------------------------------------
+
+  //@}
+
+//--==----==----==----==----==----==----==----==----==----==----==----==----==--
+
+  //============================================================================
+  ///
+  /// @defgroup cpp_kodi_addon_game_SerializationOperations 5. Serialization operations
+  /// @ingroup cpp_kodi_addon_game
+  /// @brief **Serialization operations**
+  ///
+  //@{
+
+  //============================================================================
+  ///
+  /// @brief Get the number of bytes required to serialize the game
+  ///
+  /// @return the number of bytes, or 0 if serialization is not supported
+  ///
+  virtual size_t SerializeSize()
+  {
+    return 0;
+  }
+  //----------------------------------------------------------------------------
+
+  //============================================================================
+  ///
+  /// @brief Serialize the state of the game
+  ///
+  /// @param[in] data The buffer receiving the serialized game data
+  /// @param[in] size The size of the buffer
+  ///
+  /// @return the error, or @ref GAME_ERROR_NO_ERROR if the game was serialized into the buffer
+  ///
+  virtual GAME_ERROR Serialize(uint8_t* data, size_t size)
+  {
+    return GAME_ERROR_NOT_IMPLEMENTED;
+  }
+  //----------------------------------------------------------------------------
+
+  //============================================================================
+  ///
+  /// @brief Deserialize the game from the given state
```

Comment:
> The `ColumnLimit` in `.clang-format` is set to `100` which defines line length [...]

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#3-formatting

Reply:
Do not quite understand, Sorry. You mean to increase the length if `===` to 100 there?

Reply:
Seems like GitHub shows the comment in the Conversation tab at at a wrong line. This comment is for line 2140, not for 2065.